Example/Tests/Doubles/BitmovinPlayerTestDouble.swift

```diff
@@ -15,7 +15,9 @@ class BitmovinPlayerTestDouble: BitmovinPlayer, TestDoubleDataSource {
     var fakeListener: PlayerListener?
 
     override init() {
-        super.init(configuration: PlayerConfiguration())
+        let config = PlayerConfiguration()
+        config.key = "foobar"
```

Comment:
What is the purpose of setting `config.key`?

Reply:
To be able to run the tests without modification, i.e. without manually adding a player license key.

Reply:
Understood, thanks.

Reply:
@luckygoyal-bitmovin if we don't find a player license (valid or invalid) at setup, we will throw an exception. However, in the tests, we never actually try to load a source / start playback. So we just need a non-empty string in tests to bypass the exception handling in the SDK.